theseus/geometry/vector.py

```diff
@@ -204,7 +204,21 @@ def _project_impl(
         return euclidean_grad.clone()
 
     @staticmethod
-    def exp_map(tangent_vector: torch.Tensor) -> "Vector":
+    def exp_map(
```

Comment:
We should update tests for Vector and Point2/3 as well.

Reply:
Just for sanity check and completeness :) and to make sure the compute graph didn't break anywhere which the torch.autograd would then reveal.

Reply:
Tested.